gobblin-utility/src/main/java/gobblin/util/ProxiedFileSystemWrapper.java

```diff
@@ -110,7 +110,7 @@ public FileSystem getProxiedFileSystem(State properties, AuthType authType, Stri
       @Override
       public Void run() throws IOException {
         LOG.debug("Now performing file system operations as :" + UserGroupInformation.getCurrentUser());
-        proxiedFs = FileSystem.get(fsURI, conf);
+        proxiedFs = FileSystem.newInstance(fsURI, conf);
```

Comment:
I have concerns about this, as this gets called often by other parts of Gobblin, which could impact performance. Ideally, I would want to see something like HDFS and local filesystems are not closed by the fshelper, which would allow us to use `get` in here (since proxied file systems are not generally used for filesystems that should be closed).